docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1042,6 +1042,38 @@ Status Codes:
           }
         ]
 
+** Example request, with digest information**:
```

Comment:
Remove the space between ** and the letter E as this is not resolving in display.